tests/kernel/module_build.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test kernel module build
+#          Example http://www.tldp.org/LDP/lkmpg/2.6/html/x121.html
+# Maintainer: Petr Cervinka <pcervinka@suse.com>
+# Tags: https://progress.opensuse.org/issues/49031
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    select_console "root-console";
```

Comment:
maybe we can run it in virtio console ?

Reply:
I knew that you will ask :) And now I don't have excuse as spvm or something, I will check it...

Reply:
I am not trying to push you so you don't need "excuse" . "Because I want to use root-console" - is good enough "excuse" :)

Reply:
changed to `select_serial_terminal`